docs/source/tutorials/how_to_visualize_scan_results.rst

```diff
@@ -3,109 +3,15 @@
 How to Visualize Scan results
 =============================
 
-In this simple tutorial example, we import results from a basic scan performed on the ``samples``
-directory distributed by default with Scancode, and visualize the outputs through
-Scancode Workbench.
+To help visualize the scans, we have a dedicated tool  `Scancode workbench <https://github.com/nexb/scancode-workbench/>`_  which is a desktop application that allows you to visualize and explore the results of one or more scans. It is a cross-platform application that runs on Windows, Mac OS X and Linux. It is built using the Electron framework and is built using Electron, Typescript & React
+
+Detailed Installation and Usage guide can be found here -  :ref:`scancode-workbench:getting-started`
 
 .. WARNING::
 
-    This tutorial uses the 3.1.1 version of Scancode Toolkit, and Scancode Workbench 3.1.0 (This
+    This tutorial uses the 32.x version of Scancode Toolkit, and Scancode Workbench 4.0-beta (This
```

Comment:
This can be `4.0.x` as we're almost there

Reply:
Sure, I've updated it